src/style.css

```diff
@@ -0,0 +1,137 @@
+html {
+  font-family: "Roboto", sans-serif;
+
+  --color-button: #00acdc;
+  --color-main-text: #060b35;
+  --horizontal-padding: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 198px;
```

Comment:
Please use width 200px and box-sizing: border-box to include border-width to general element width